template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/validation/SafetyValveValidator.java

```diff
@@ -0,0 +1,40 @@
+package com.sequenceiq.cloudbreak.cmtemplate.validation;
+
+import java.util.List;
+
+import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.dataformat.xml.XmlMapper;
+import com.sequenceiq.cloudbreak.template.model.SafetyValve;
+
+@Component
+public class SafetyValveValidator {
```

Comment:
Does it have any benefits of making this a Spring bean? Why not move the class over to `src/test/java`? In the latter case, `core` should have a test dependency for `template-manager-cmtemplate`:
```
testImplementation project(path: ':template-manager-cmtemplate', configuration: 'tests')
```

Reply:
I am going to reuse it to execute validations when safety valve value is modified via ConfigMapper.

Reply:
Ah okay, makes sense then. :smile: